sci-electronics/qelectrotech/qelectrotech-0.90.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2001-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8?

Reply:
Updated in https://github.com/gentoo/gentoo/commit/d49beac3d6715a7082989331ad295232b8be9410
had to replace xdg_src_prepare by eapply_user in src_prepare